src/PKSim.Core/Snapshots/Mappers/CompoundMapper.cs

```diff
@@ -158,7 +158,7 @@ private IEnumerable<IParameter> changedGroupParameters(IContainer container, str
       {
          var alternativeGroup = compound.ParameterAlternativeGroup(alternativeGroupName);
          var alteratives = await _alternativeMapper.MapToSnapshots(alternativeGroup.AllAlternatives);
-         var definedAlternatives = alteratives?.Where(x => x != null).ToArray();
+         var definedAlternatives = alteratives?.ToArray();
```

Comment:
This is not required anymore. I have changed the MapToSnapshots to NOT return any null object